### PR TITLE
Implement Error screen layout according to mockup

### PR DESCRIPTION
**Pull request by diegchav**
Create Error component with layout from mockup and add it to App.js. Currently the component is commented out and the 'Go back' icon doesn't do anything.

### CODE REVIEW

File: src/components/Error/Error.module.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.errorScreen {
+  position: absolute;
+  width: 100vw;
+  height: 100%;
+  background: #000;
```
Comment (ArnasDickus) [inline src/components/Error/Error.module.scss]: The idea, is if images can't be shared this popup appears. So you don't need to add black background.

File: src/components/Error/Error.module.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  align-items: center;
+  justify-content: center;
+  background: linear-gradient(180deg, #FB3600 0%, rgba(251, 54, 0, 0.47) 100%);
+
+  .guitarIcon {
+    margin-bottom: 50px;
+  }
+
+  .errorMessage {
+    font-family: Biryani;
```
Comment (ArnasDickus) [inline src/components/Error/Error.module.scss]: Biryani is already added in global scss file. You don't need to add it here.

File: src/components/Error/Error.module.scss [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  width: 100vw;
+  height: 100%;
+  background: #000;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.errorModal {
+  width: 450px;
```
Comment (ArnasDickus) [inline src/components/Error/Error.module.scss]: Use rem, em over pixels.

File: src/components/Error/Error.module.scss [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+  .goBackIcon {
+    margin-top: 30px;
+    cursor: pointer;
+  }
+}
+
+@media only screen and (max-width: 480px) {
+  .errorModal {
+    width: 300px;
```
Comment (ArnasDickus) [inline src/components/Error/Error.module.scss]: Use rem, em over pixels.

File: src/components/Error/Error.module.scss [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    height: 450px;
+
+    .guitarIcon {
+      margin-bottom: 30px;
```
Comment (ArnasDickus) [inline src/components/Error/Error.module.scss]: Use rem, em over pixels.

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello, thanks for adding a PR. Here's my concerns:

1) Format your code by running `npm run format`
2) Increase Z-index popup should be on top.

File: src/components/Error/Error.module.scss (hunk 1 above)
Comment (diegchav) [reply src/components/Error/Error.module.scss]: It makes, I wasn't sure about the background. I'm gonna make all the changes.

**Comment by ArnasDickus** [comment]
Hello @diegchav so when are you planning on adding changes to this task?

**Comment by diegchav** [comment]
Hi @ArnasDickus I made the requested changes.

**Review by ArnasDickus — APPROVED** [review]
Hello, thanks for fixing all my PR issues. Looks good, although that red color is a little too bright for me, that's another discussion with a designer. Good work.

**Pull request merged**